src/main/java/edu/harvard/iq/dataverse/util/StringUtil.java

```diff
@@ -180,6 +180,19 @@ public static String decrypt(String value, String password ) {
         }
     }
     
+    public static String sanitizeFileDirectory(String value){
+        
+        while (value.startsWith("\\") || value.startsWith("/") || value.startsWith("-") || value.startsWith(".")){
+            value = value.substring(1);
+        }
+        while (value.endsWith("\\") || value.endsWith("/") || value.endsWith("-") || value.endsWith(".")){
+            value = value.substring(0, value.length() - 1);
+        }
+        
```

Comment:
@sekmiller This method correctly strips the leading and trailing slashes (and also "." and "-"); 
But I thought the plan was also to replace any multiple slashes between nested folders with a single slash. 
For example, as implemented now, I can enter "folder1///folder2", and it gets saved and displayed like this, with the 3 slashes. 
I thought we wanted to sanitize it as "folder1/folder2".

Reply:
Added a remove of multiple inner slashes to the sanitizer as suggested. Thanks @landreev